src/py_pecg/Subfunction_Module.py

```diff
@@ -0,0 +1,332 @@
+import numpy as np
+
+
+def buscamin(x: np.ndarray) -> int:
+    """
+    This function searches the first local minimum of the modulus of an array x,
+    While truncating the edges (First and Final Indices).
+
+    Args:
+        x (np.ndarray): The target array.
+
+    Returns:
+        ind (int): The first index in x for which the first minimum is located.
+
+    Credits:
+        Original authors: Juan Pablo Martínez, Rute Almeida, Salvador Olmos,
+        Ana Paula Rocha, Pablo Laguna
+        Original Publication: A wavelet-based ECG delineator:
+        evaluation on standard databases
+        Original DOI: 10.1109/TBME.2003.821031.
+    """
+    x = np.abs(x)
+    localmin = (x[1:-1] <= x[:-2]) & (x[1:-1] <= x[2:])
+    ind = np.argmax(localmin)
```

Comment:
Using `argmax()` on a Boolean array is somewhat risky. Although NumPy contributors chosen that:
```
In case of multiple occurrences of the maximum values, the indices corresponding to the first occurrence are returned.
```
This is a rather random choice and people need to know this implementation detail of  `np.argmax()` to understand your code.

I'd recommend using NumPy functions that are intended to conditionally find an index like:
  ```suggestion
    ind = np.argwhere(localmin)[0]
```
or
  ```suggestion
    ind = min(np.argwhere(localmin))
```